homeassistant/components/proxmoxve/__init__.py

```diff
@@ -188,7 +189,7 @@ def build_client() -> ProxmoxAPI:
 
 
 def create_coordinator_container_vm(
-    hass, proxmox, host_name, node_name, vm_id, vm_type
+    hass: HomeAssistant, proxmox, host_name, node_name, vm_id, vm_type
```

Comment:
It would be good to add type hints to all arguments, as well as the return type.